resource-timing/resources/webperftestharness.js

```diff
@@ -107,7 +107,7 @@ function test_timing_greater_than(attribute_name, greater_than, properties)
 function test_timing_equals(attribute_name, equals, msg, properties)
 {
     var test_msg = msg || "window.performance.timing." + attribute_name + " == " + equals;
-    test_equals(performanceNamespace.timing[attribute_name], equals, test_msg, properties);
+    assert_equals(performanceNamespace.timing[attribute_name], equals, test_msg, properties);
```

Comment:
This assert might not be inside any test, doesn't that break any test when the assert fails?